holoviews/plotting/mpl/stats.py

```diff
@@ -204,3 +205,49 @@ def teardown_handles(self):
         for group in self.handles['box'].values():
             for v in group:
                 v.remove()
+
+
+class HexTilesPlot(ColorbarPlot):
```

Comment:
Why is this in ``stats.py`` when the bokeh equivalent is in ``hex_tiles.py``?

Reply:
Can create a new file here too, didn't seem worth it though given that it's fairly short and doesn't have to implement the aggregation/tiling code. Happy to move it though.

Reply:
I would prefer consistency in structure when the elements supported match across backends.

Reply:
Sounds good, I'll move it.